trunner/target/armv7a7.py

```diff
@@ -84,6 +84,15 @@ class IMX6ULLEvkTarget(ARMv7A7Target):
             block_size=0x10000,
             cleanmarker_size=0x10,
         ),
+        # Setup of neutral block erase timeout
+        # This approach was led by documentation of MT25QL256 IC Flash.
+        # On system is declared 64kb block size but to erase is used 4kb
+        # subsector erase, in order to create average, is needed to divide
+        # block size by sector erase to achieve 16 erases on block and simply
+        # multiply that by subsector erase time. The time achieved by this
+        # calculation is the maximal possible erase time. To get average,
+        # is needed to divide that value by 2.
```

Comment:
I suggest a bit shorter comment, but showing why such value has been set:

```suggestion
        # Based on MT25QL256 IC docs 4KB subsector erase time equals 0.4s max.
        # Blocksize set for jffs2 command is 64KB, so 16 sectors have to be erased for 1 block
        # That's why we set (0.4 * 16 / 2) as a timeout. The value is divided by 2 to early detect slowing the flash
```

Please do sth similar for the second target.